src/alert_rules/prometheus/prometheus_alerts.yaml

```
@@ -138,3 +138,13 @@
     "for": "5m"
     "labels":
       "severity": "warning"
+
+  - "alert": "OpenSearchThrottlingTooLong"
+    "annotations":
+      "message": "Cluster {{ $labels.cluster }} is throttling for two hours. Please optimize queries and indexing patterns or consider scale the application."
```

I would recommend a slightly different text. Generally, when we get into this situation, it means the indexing request rate is higher than we anticipated OR we did not *planned* this index very well. For the later, something like lifecycle management and breaking the index into different bits (e.g. a time based index ingesting logs can be broken into a per-day index) would be more appropriate.

```suggestion
      "message": "Cluster {{ $labels.cluster }} is throttling for at least two hours. Please review your indexing request rate, index lifecycle or consider scale the application."
```